crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -149,10 +142,7 @@ contract OnChainProposer is IOnChainProposer, ReentrancyGuard {
         bytes32 depositLogs
     ) external override onlySequencer {
         require(
-            blockNumber == nextBlockToCommit ||
-                (blockNumber == 0 &&
-                    lastCommittedBlock == 0 &&
-                    nextBlockToCommit == 0),
+            blockNumber == lastCommittedBlock + 1,
```

Comment:
Isn't `blockNumber == 0` still a valid case?

Reply:
`blockNumber == 0` would be trying to commit the genesis. We decided to not commit nor prove the genesis